### ISSUE TITLE
Add info on number of Palette splices

### ISSUE THREAD

**Issue by MimbaMonkeyHouse** [gdombiak/OctoPod#264]
It would be great to have information on the number of splices already done and the total number of splices for the current print directly from OctoPod.

**Comment by gdombiak** [comment]
Hi @MimbaMonkeyHouse,

Maybe I'm misunderstanding the request, is the information you are requesting different from what appears in the second row of the Palette 2 section?

![Screen Shot 2020-02-10 at 7 22 07 PM](https://user-images.githubusercontent.com/1791267/74208960-a2b7f380-4c3a-11ea-927d-71b66dc1c146.png)

**Comment by MimbaMonkeyHouse** [comment]
Hey @gdombiak. Sorry, I'm a monkey and missed this.
I'm trying to get info from Palette to use in Home Assitant. I can get the pings by using:
```
$.ajax({
                url: API_BASEURL + "plugin/palette2",
                type: "POST",
                dataType: "json",
                data: JSON.stringify({
                    command: "getPingHistory"
                }),
                contentType: "application/json; charset=UTF-8"
            }).done(function(data){console.log(data)});
```
This gives the entire history rather than just the last Ping that I need.
Can you let me know how you get the Splice (total and current), pings and pongs fro mthe Palette Plugin?

**Comment by gdombiak** [comment]
Hey @MimbaMonkeyHouse (monkey) :)

OctoPod executes the command **uiUpdate** which is the same thing that the plugin does in the OctoPrint's UI. However, that HTTP response does not include the data. Instead the plugin will write back via the websocket and pass all the data that you see in the OctoPrint's UI. OctoPod then parses the JSON that came back in the websocket to extract the splices and the rest of the information.

I had to add **getPingHistory** for Siri support and some stats that I show in the UI and since I added that PR, I include the data in the HTTP response (unlike how the **uiUpdate** command works)
